lib/kiev/rack/request_logger.rb

```diff
@@ -115,6 +115,15 @@ def form_data(request:, began_at:, status:, env:, body:, response:, exception:)
             full_body << str
           end
           data[:body] = full_body.join
+          if data[:body] && !data[:body].empty? && response.headers["Content-Encoding"] == "gzip"
+            begin
+              sio = StringIO.new(data[:body])
+              gz = Zlib::GzipReader.new(sio)
```

Comment:
I'm wondering why we don't have to `require 'zlib'` here. Is this implicitly loaded by another gem / require statement already? If yes, shall we add it explicitly here too so we are not betting on the implicit `include'`?

Reply:
I added the explicit require